.github/workflows/ci.yml

```diff
@@ -8,19 +8,36 @@ on:
 
 jobs:
   tests:
-    name: "Tests ${{ matrix.php-version }} deps ${{ matrix.dependency-versions }}"
     runs-on: ubuntu-22.04
 
     strategy:
       fail-fast: false
       matrix:
-        # normal, highest, non-dev installs
         php-version: ['8.1', '8.2', '8.3', '8.4']
-        dependency-versions: ['highest']
+        dependency-version: [ '' ]
+        symfony-version: [ '' ]
         include:
-          # testing lowest PHP version with lowest dependencies
-          - php-version: '8.1'
-            dependency-versions: 'lowest'
+          # dev packages (probably not needed to have multiple such jobs)
+          -   minimum-stability: 'dev'
+              php-version: '8.4'
+          # lowest deps
+          -   php-version: '8.1'
+              dependency-version: 'lowest'
+          # LTS version of Symfony
+          -   php-version: '8.1'
+              symfony-version: '6.4.*'
+          # Explicit Symfony versions
+          -  php-version: '8.1'
+             symfony-version: '5.4'
+          -  php-version: '8.1'
+             symfony-version: '6.2'
```

Comment:
I wouldn't waste resources on EOL versions

Reply:
I used 6.2 and 7.0 because the Bundle support these versions. I also tend to not really care about unmaintained versions, but is it fine here?

Reply:
I'm totally fine with that, but I was thinking doing that in a second PR, and then release WebpackEncoreBundle 3.0